docs/networkpolicy-recommendation.md

```diff
@@ -0,0 +1,148 @@
+in the cluster. Our policy recommendation logic is implemented as a [Spark](https://github.com/apache/spark)
+application, the Kubernetes Operator for Apache Spark helps us schedule the Spark job in the Kubernetes cluster.
+
+Please run the following commands:
+
+```bash
+helm repo add spark-operator https://googlecloudplatform.github.io/spark-on-k8s-operator
+helm install policy-reco spark-operator/spark-operator --namespace flow-visibility --set image.tag=latest
+```
+
+This will install the Kubernetes Operator for Apache Spark into the `flow-visibility` Namespace.
+
+Once we finish these deployment steps, we should see the `policy-reco-spark-operator` Pod running inside the `flow-visibility` namespace:
+
+```bash
+$ kubectl get pods -n flow-visibility
+NAME                                          READY   STATUS    RESTARTS   AGE
+policy-reco-spark-operator-56c4cb454c-4vhfh   1/1     Running   0          2m19s
+``` -->
+
+## Perform NetworkPolicy Recommendation
+
+Currently, the NetworkPolicy recommendation feature only supports command line interaction through `theia`.
+`theia` is the command-line tool which provides access to Theia network flow visibility capabilities.
+To get more information about `theia`, please refer to this [doc](theia.md).
+
+We have 3 `theia` commands for the policy recommendation feature:
+
+- `theia policy-recommendation run`
+- `theia policy-recommendation status`
+- `theia policy-recommendation retrieve`
+
+Or you could use `pr` as a short alias of `policy-recommendation`:
+
+- `theia pr run`
+- `theia pr status`
+- `theia pr retrieve`
+
+To see all options and usage examples of these commands, you may run `theia policy-recommendation [subcommand] --help`.
+
+### Run a policy recommendation job
+
+The `theia policy-recommendation run` command can run a new policy recommendation job.
+If the new policy recommendation job is created successfully, the `recommendation ID` of this job will be returned:
+
+```bash
+$ theia policy-recommendation run
+Successfully created policy recommendation job with ID e998433e-accb-4888-9fc8-06563f073e86
+```
+
+A policy recommendation job may take a few minutes to an hour to complete
+depending on the number of network flows. To run a policy recommendation job
+and wait until it is finished run:
+
+```bash
+theia policy-recommendation run --wait
+```
+
+### Check the status of a policy recommendation job
+
+The `theia policy-recommendation status` command is used to check the status of a previous policy recommendation job.
+
+For example the job we just created above, we could check the status of it by:
+
+```bash
+$ theia policy-recommendation status e998433e-accb-4888-9fc8-06563f073e86
+Status of this policy recommendation job is COMPLETED
+```
+
+It will return the status of this policy recommendation job like `SUBMITTED`, `RUNNING`, `COMPLETED`, and `FAILED`.
```

Comment:
"It will return the status of this policy recommendation job like" -> "It will return the status of this policy recommendation job, which can be one of `SUBMITTED`, `RUNNING`, `COMPLETED`, and `FAILED`" ?

Reply:
Thanks, I added `etc` at the end since there are more possible statuses.